gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CgmesExportService.java

```diff
@@ -109,58 +102,40 @@ Map<String, ByteArrayOutputStream> createAllSshFiles(Network mergedNetwork, SweD
             Country country = entry.getKey();
             String tso = entry.getValue();
             if (subnetworksByCountry.containsKey(country)) {
-                mapSshFiles.putAll(createOneSsh(subnetworksByCountry.get(country), sweData, tso));
+                LOGGER.info("Building cgmes files for country {}", country);
+                mapFiles.putAll(createTsoFiles(subnetworksByCountry.get(country), sweData, tso));
             }
         }
-        return mapSshFiles;
+        return mapFiles;
     }
 
-    private Map<String, ByteArrayOutputStream> retrieveEqAndTpFiles(SweData sweData) throws IOException {
-        LOGGER.info("Retrieving EQ & TP files");
+    private Map<String, ByteArrayOutputStream> createTsoFiles(Network network, SweData sweData, String tso) throws IOException {
         Map<String, ByteArrayOutputStream> mapFiles = new HashMap<>();
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.RTE_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REE_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REN_TP));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.RTE_EQ));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REE_EQ));
-        mapFiles.putAll(createOneFile(sweData, CgmesFileType.REN_EQ));
+        Properties exportParams = new Properties();
+        exportParams.put(CgmesExport.PROFILES, List.of("EQ", "TP", "SSH"));
```

Comment:
One possible improvement would be to define the `List.of("EQ", "TP", "SSH")` as a constant and use it to replace lines 119-121 below with a for loop:
```java
List<String> CGMES_FILETYPES = List.of("EQ", "TP", "SSH");

[...]

network.write("CGMES", exportParams, memDataSource);
for (String type : CGMES_FILETYPES) {
    putAndRenameFile(network.getNameOrId(), sweData, tso, memDataSource, mapFiles, type);
}
```